docs/import/manage_disaster.html

```diff
@@ -120,7 +120,7 @@
       </div>
       <div><label for="damage-asset-select">Damage asset <a href="https://docs.google.com/document/d/1WnjHIexMnlVkn5lqnzYlBVg4aWxh2wVDZI2DBAu-TGI/edit?ts=5de7cd32#bookmark=id.ppvphy5dof7n" target="_blank"><i class="help fa fa-question-circle" aria-hidden="true"></i></a>: </label><select id="damage-asset-select"></select>
         <div id="map-bounds-div" hidden>
-         OR (before damage asset available)<a href="https://docs.google.com/document/d/1WnjHIexMnlVkn5lqnzYlBVg4aWxh2wVDZI2DBAu-TGI/edit?ts=5de7cd32#bookmark=id.pmr206t53gos" target="_blank"><i class="help fa fa-question-circle" aria-hidden="true"></i></a>:
+         OR (before damage asset available) draw a region<a href="https://docs.google.com/document/d/1WnjHIexMnlVkn5lqnzYlBVg4aWxh2wVDZI2DBAu-TGI/edit?ts=5de7cd32#bookmark=id.pmr206t53gos" target="_blank"><i class="help fa fa-question-circle" aria-hidden="true"></i></a>:
```

Comment:
nit: sort of strange how the font for 'Damage Asset' is so different from the rest of the font - maybe because it's a label? maybe also make this text a label?

Reply:
Agree, but I don't think it makes sense to make this text a label for the map div. Better to either make label text less different or make damage asset not a label. Made it not a label for now.